src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/NewRelic.Agent.Extensions.csproj

```diff
@@ -17,6 +17,9 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(RootProjectDirectory)\src\NewRelic.Core\NewRelic.Core.csproj" />
+  </ItemGroup>
```

Comment:
Does this introduce a circular dependency between NewRelic.Core.csproj and NewRelic.Agent.Extensions.csproj?

Reply:
It does not. `NewRelic.Core` has no project dependencies.